pages/app/my-profile.js

```diff
@@ -1,9 +1,30 @@
-// Profile View
-import { Image } from 'next/image';
+import { useState, useEffect } from 'react';
+import { Avatar } from '@mui/material';
+import styles from '../../utils/styles/Profiles.module.css';
+import defaultProfilePic from '../../utils/constants';
+
+// word-wrap normal in the global page
+// add user type: teacher or student
+export default function MyProfile(props) {
+  const [userType, setUserType] = useState('mentor');
 
-export default function MyProfile() {
   return (
-    <div className='pageData'>My Profile</div>
+    <div className="pageData">
+      <div className={styles.profileContainer}>
+          <div className={styles.profilePic}>
+            <Avatar
+              className="my-profile-view-avatar"
+              alt="My Profile Picture"
+              src={defaultProfilePic}
+              sx={{ width: 300, height: 300 }}
```

Comment:
I'm thinking we add default dimensions for things like profile pics to constants file. Lmk your thoughts on this